spacy/cli/debug_diff.py

```diff
@@ -0,0 +1,86 @@
+from typing import Optional
+
+import typer
+from wasabi import Printer, diff_strings, MarkdownRenderer
+from pathlib import Path
+from thinc.api import Config
+
+from ._util import debug_cli, Arg, Opt, show_validation_error, parse_config_overrides
+from ..util import load_config
+from .init_config import init_config, Optimizations
+
+
+@debug_cli.command(
+    "diff",
```

Comment:
I wonder whether we should call this `diff-config`, similar to how the `init` subcommand is called `fill-config`? Would love to hear other's opinions on this first though.

Reply:
I also share some confusion about what `debug diff` might be diff-ing. `diff-config` doesn't sound too bad, although I wonder a teensy bit whether `diff` is understandable enough. `compare-config`? Hmm...

Reply:
Ah, I think `diff-config` sounds good! Just to parallel `fill-config`. Going back I realized that `diff` may sound too general and it can mean diffing other things. `diff-config` sounds more explicit. Can change that 👍

Reply:
Updated here: https://github.com/explosion/spaCy/pull/10502/commits/79d0aa526770298777192e7e492e877042cf6fe6